packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -289,6 +289,25 @@ export interface IButtonProps
    */
   persistMenu?: boolean;
 
+  /**
+   * If true, the persisted menu is rendered hidden when the button
```

Comment:
would like a note about why this is already deprecated. sounds like this flag turns on experimental support for faster 'persistMenu' that isn't supported by edge yet.

Reply:
almost makes me want an 'experimental' type prop